.travis.yml

```diff
@@ -2,18 +2,24 @@ language: node_js
 sudo: required
 notifications:
   email: false
+  slack:
```

Comment:
Are we sending notification to any Slack room with this configuration?

Reply:
@mrizzi  Slack notifications were added into [insights-frontend-starter-app](https://github.com/RedHatInsights/insights-frontend-starter-app) using this commit https://github.com/RedHatInsights/insights-frontend-starter-app/commit/cd769f585f517fb21a4d366fce55abccdc77aec4. I'm not sure about their intention  of sending notifications to their slack channel but just in case I've just removed this part from our .travis.yml file

Reply:
Thanks for removing it